chromadb/api/segment.py

```diff
@@ -685,7 +685,7 @@ def _query(
         where_document = (
             validate_where_document(where_document)
             if where_document is not None and len(where_document) > 0
-            else where_document
+            else None
```

Comment:
Do we need this change? Are we changing the default from {} to None? If so we should do that extensively